libarchive/archive_write_set_format_iso9660.c

```diff
@@ -2553,7 +2553,7 @@ set_date_time(unsigned char *p, time_t t)
 static void
 set_date_time_null(unsigned char *p)
 {
-	memset(p, '0', 16);
+	memset(p, (int)'0', 16);
```

Comment:
Sounds like something to fill a bug report about. There really is absolutely no reason for the cast.